src/web/wms/src/main/java/org/geoserver/wms/web/data/AbstractStylePage.java

```diff
@@ -45,14 +46,18 @@
 import org.geoserver.web.ComponentAuthorizer;
 import org.geoserver.web.GeoServerApplication;
 import org.geoserver.web.GeoServerSecuredPage;
-import org.geoserver.web.data.style.StyleDetachableModel;
 import org.geoserver.web.wicket.CodeMirrorEditor;
 import org.geoserver.web.wicket.GeoServerAjaxFormLink;
 import org.geoserver.web.wicket.ParamResourceModel;
 import org.xml.sax.SAXParseException;
 
 /**
  * Base page for creating/editing styles
+ * <p>
+ * WARNING: one crucial aspect of this page is its ability to not loose edits when one switches from
```

Comment:
Thats fair, regular comment probably would have been fine. PublishedConfigurationPage does the same thing, so I guess this is more consistent?